app/views/qsets/_modal_new_qset.html.erb

```diff
@@ -14,6 +14,12 @@
             <input type="text" class="form-control" name="name" />
           </div>
           <div class="form-group">
+            <label for="text">Qset Type: </label>
+            Questions<input type="radio" name="qset_type" value="questions" checked>
+            Subsets<input type="radio" name="qset_type" value="subsets">
+            Mixed<input type="radio" name="qset_type" value="mixed">
+          </div>
```

Comment:
I note that if you cancel the dialog and re-open it, the new qset name is reset/blanked out, but this qset type value is not reset. We should probably do that, for consistency. See https://github.com/penzance/askup/blob/master/app/assets/javascripts/qsets.js#L11